ui/app/AppLayouts/Chat/views/communities/qmldir

```diff
@@ -1,7 +1,8 @@
+ChannelsSelectionModel 1.0 ChannelsSelectionModel.qml
+CommunityNewCollectibleView 1.0 CommunityNewCollectibleView.qml
 CommunityNewPermissionView 1.0 CommunityNewPermissionView.qml
 CommunityPermissionsView 1.0 CommunityPermissionsView.qml
 CommunityWelcomePermissionsView 1.0 CommunityWelcomePermissionsView.qml
```

Comment:
I think this is a leftover of one of my lasts prs.. Could you kindly remove this line? `CommunityWelcomePermissionsView` was removed and created `CommunityWelcomeSettingsView` instead! Thanks! :)